isotpsend.c

```diff
@@ -178,6 +182,22 @@ int main(int argc, char **argv)
 		    }
 		    break;
 
+	    case 'l':
+		    if (optarg[0] == 'i') {
+			    loops = 0; /* infinite loop */
+		    } else {
+			    loops = strtoul(optarg, (char **)NULL, 10);
```

Comment:
`(char **)` cast not needed

Reply:
Ugh. I have tons(!) of these casts in the can-utils code. Should I create an additional patch to remove them entirely?